dwave/system/package_info.py

```diff
@@ -19,3 +19,18 @@
 __author__ = 'D-Wave Systems Inc.'
 __authoremail__ = 'acondello@dwavesys.com'
 __description__ = 'All things D-Wave System.'
+
+
+# register the non-open-source packages required for dwave-drivers to work
+contrib = [{
+    'name': 'drivers',
+    'title': 'D-Wave Drivers',
+    'description': 'These drivers enable some automated performance-tuning features.',
+    'license': {
+        'name': 'D-Wave EULA',
+        'url': 'https://docs.ocean.dwavesys.com/eula',
+    },
+    'requirements': [
+        'dwave-drivers>=0.4.4',
```

Comment:
might want to add a comment in setup.py as a reminder to keep this updated

Reply:
Added a note to `setup.py`.